add-custom.sh

```
@@ -12,7 +12,7 @@ function usage() {
 
 binary_download_url=
 name=custom
-java_version=8
+java_version=${DEFAULT_JAVA}
```

Moving this initialization below the while loop using the following code would make the script more robust:
```
if [ -z "${java_version}" ]; then
  [ -z "${DEFAULT_JAVA}" ] && echo "Neither -j nor DEFAULT_JAVA are specified." && exit 1
  java_version="${DEFAULT_JAVA}"
fi
```

---

Shouldn't we rather fail in this very line if DEFAULT_JAVA is not set? And that you could achieved with `-u`.